tests/test_trade.py

```diff
@@ -56,13 +56,19 @@ def run_base_trade_test(self, policy):
             "fee_percent": 0.1,
             "init_pool_apy": 0.05,
         }
+        if additional_overrides:
```

Comment:
Can you specify it in the same way as override_dict, and then use override_dict.update(additional_overrides)?

Reply:
cleaned it up, commented below where it's used